pkg/config/replica_config_test.go

```diff
@@ -77,9 +77,9 @@ func TestReplicaConfigOutDated(t *testing.T) {
 	conf.Mounter.WorkerNum = 3
 	conf.Sink.Protocol = "open-protocol"
 	conf.Sink.DispatchRules = []*DispatchRule{
-		{Matcher: []string{"a.b"}, Dispatcher: "r1"},
-		{Matcher: []string{"a.c"}, Dispatcher: "r2"},
-		{Matcher: []string{"a.d"}, Dispatcher: "r2"},
+		{Matcher: []string{"a.b"}, PartitionRule: "r1"},
```

Comment:
Several test files related to `Sink.DispatchRules` should also be modified. I will add later.